cmd/drand-cli/cli.go

```diff
@@ -278,7 +278,7 @@ var appCommands = []*cli.Command{
 		Name: "generate-keypair",
 		Usage: "Generate the longterm keypair (drand.private, drand.public)" +
 			"for this node.\n",
-		ArgsUsage: "<address> is the public address for other nodes to contact",
+		ArgsUsage: "<address> is the address for other nodes to contact (specified as 'private-listen' to the daemon)",
```

Comment:
I read this as though the `generate-keypair` contacts other nodes for some reason but I believe the meaning is actually that other nodes can contact this node on the address specified.

```suggestion
		ArgsUsage: "<address> is the address other nodes will be able to contact this node on (specified as 'private-listen' to the daemon)",
```